channel.c

```diff
@@ -51,6 +51,11 @@ static const char * const iio_chan_type_name_spec[] = {
 	[IIO_CONCENTRATION] = "concentration",
 	[IIO_RESISTANCE] = "resistance",
 	[IIO_PH] = "ph",
+	[IIO_UVINDEX] = "uvindex",
+	[IIO_ELECTRICALCONDUCTIVITY] = "electricalconductivity",
+	[IIO_COUNT] = "count",
+	[IIO_INDEX] = "index",
+	[IIO_GRAVITY] = "gravity",
```

Comment:
Hi,

Thanks for the patch. I think we also need to update the iio_chan_type and iio_modifier enums in iio.h

Reply:
Hi @lclausen-adi, indeed, sorry about that. I'll amend it.